### PR TITLE
fix(ci): disable trivy-scan-image

### LINKED ISSUE #3459: github/codeql-action/upload-sarif - JavaScript heap out of memory
I started noticing this issue today. Our CodeQL results aren't being uploaded due to a memory issue.

See [logs](https://github.com/cdr/code-server/pull/3450/checks?check_run_id=2655525849#step:6:1)
```logs
Run github/codeql-action/upload-sarif@v1
Uploading sarif files: ["trivy-image-results.sarif"]
FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - JavaScript heap out of memory

<--- Last few GCs --->

[2047:0x42960e0]   211606 ms: Mark-sweep 1968.4 (1973.9) -> 1968.4 (1970.9) MB, 1146.9 / 0.0 ms  (average mu = 0.902, current mu = 0.000) last resort GC in old space requested
[2047:0x42960e0]   212790 ms: Mark-sweep 1968.4 (1970.9) -> 1968.4 (1970.9) MB, 1183.6 / 0.0 ms  (average mu = 0.811, current mu = 0.000) last resort GC in old space requested


<--- JS stacktrace --->

==== JS stack trace =========================================

    0: ExitFrame [pc: 0x1374fd9]
Security context: 0x1df0d8ac08a1 <JSObject>
    1: outputHash(aka outputHash) [0x23868395bba1] [/home/runner/work/_actions/github/codeql-action/v1/lib/fingerprints.js:~63] [pc=0x4e7bc427b85](this=0x2d8a3d4404a9 <undefined>)
    2: hash(aka hash) [0x4b2344f8471] [/home/runner/work/_actions/github/codeql-action/v1/lib/fingerprints.js:~41] [pc=0x4e7bc42400b](this=0x2d8a3d4404a9 <undefined>,0x23868395bbe1 <JSF...

 1: 0x9da7c0 node::Abort() [/home/runner/runners/2.278.0/externals/node12/bin/node]
 2: 0x9db976 node::OnFatalError(char const*, char const*) [/home/runner/runners/2.278.0/externals/node12/bin/node]
 3: 0xb39f1e v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool) [/home/runner/runners/2.278.0/externals/node12/bin/node]
 4: 0xb3a299 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, bool) [/home/runner/runners/2.278.0/externals/node12/bin/node]
 5: 0xce5635  [/home/runner/runners/2.278.0/externals/node12/bin/node]
 6: 0xcf54fd v8::internal::Heap::AllocateRawWithRetryOrFail(int, v8::internal::AllocationType, v8::internal::AllocationAlignment) [/home/runner/runners/2.278.0/externals/node12/bin/node]
 7: 0xcbc118 v8::internal::Factory::NewFixedArrayWithFiller(v8::internal::RootIndex, int, v8::internal::Object, v8::internal::AllocationType) [/home/runner/runners/2.278.0/externals/node12/bin/node]
 8: 0xcbc220 v8::internal::Handle<v8::internal::FixedArray> v8::internal::Factory::NewFixedArrayWithMap<v8::internal::FixedArray>(v8::internal::RootIndex, int, v8::internal::AllocationType) [/home/runner/runners/2.278.0/externals/node12/bin/node]
 9: 0xecb760 v8::internal::HashTable<v8::internal::StringTable, v8::internal::StringTableShape>::NewInternal(v8::internal::Isolate*, int, v8::internal::AllocationType) [/home/runner/runners/2.278.0/externals/node12/bin/node]
10: 0xecb7be v8::internal::HashTable<v8::internal::StringTable, v8::internal::StringTableShape>::New(v8::internal::Isolate*, int, v8::internal::AllocationType, v8::internal::MinimumCapacity) [/home/runner/runners/2.278.0/externals/node12/bin/node]
11: 0xecbe8b v8::internal::HashTable<v8::internal::StringTable, v8::internal::StringTableShape>::EnsureCapacity(v8::internal::Isolate*, v8::internal::Handle<v8::internal::StringTable>, int, v8::internal::AllocationType) [/home/runner/runners/2.278.0/externals/node12/bin/node]
12: 0xed4af1 v8::internal::Handle<v8::internal::String> v8::internal::StringTable::LookupKey<v8::internal::InternalizedStringKey>(v8::internal::Isolate*, v8::internal::InternalizedStringKey*) [/home/runner/runners/2.278.0/externals/node12/bin/node]
13: 0xed4be6 v8::internal::StringTable::LookupString(v8::internal::Isolate*, v8::internal::Handle<v8::internal::String>) [/home/runner/runners/2.278.0/externals/node12/bin/node]
14: 0x100b30b v8::internal::Runtime_GetProperty(int, unsigned long*, v8::internal::Isolate*) [/home/runner/runners/2.278.0/externals/node12/bin/node]
15: 0x1374fd9  [/home/runner/runners/2.278.0/externals/node12/bin/node]
```

Related:
- https://github.com/github/codeql-action/issues/245#issuecomment-847191284
- https://github.com/github/codeql-action/issues/528

### PR DESCRIPTION
**Pull request by jsjoeio**
This PR temporarily disables `trivy-scan-image` due to an upstream issue. See: https://github.com/github/codeql-action/issues/528

There was a vulnerability found when CI ran. This is fixed as well:
- `browserslist`

## Checklist

- [ ] updated `CHANGELOG.md`

Fixes #3459

### CODE REVIEW

**Review by jawnsy — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -80,6 +80,7 @@
     "doctoc/**/trim": "^1.0.0",
     "postcss": "^8.2.1",
     "parcel-bundler/cssnano": "^5.0.2",
+    "browserslist": "^4.16.5",
```
Comment (jawnsy) [inline package.json:83]: I think it's best to keep unrelated changes in separate PRs, otherwise the commit message is misleading

Comment (jsjoeio) [reply package.json:83]: I generally agree. But CI was failing for this PR. For fixing vulnerability updates, should we merge even if CI is failing and then submit a follow-up PR?
